### DISCUSSION TITLE
Analyzing of other Analog Meter Gaugs

### DISCUSSION THREAD

**Discussion by ArminArmout** [jomjol/AI-on-the-edge-device#1415] [Q&A] [answered]
Hi Guy's,

first of all this is a really nice and great  project. 
Put after buying and installing the needed equipment i realized that this Project is only supporting readings from Water Meters.
Is there maybe a solution of reading other Analog Gauge Meters like this one i have attached below?
This is a Meter which shows the current filling level of my liquid Gas Tank.

Instant of checking this every week id like to get a solution which reads the filling level once per Day and transmit this to my Smart Home system.

![image](https://user-images.githubusercontent.com/113033076/204123242-19883813-bfec-49ca-bb98-d5c8fcf1b5b7.png)

Maybe you are already working on such a solution or you have an idea how i could solve this issue.

Thank you in advance.

**Comment by jomjol** [comment, marked as answer]
This will not work with the current setup and we are not working on it.
In principle the algorithm is capable of doing so. 
BUT: 
1) That would need training data for the neural network of 1-2 image for each possible position of the needle.
2) A algo that needs to be extended to not 360° needle, but only part rotation needles including the corresponding scale

Effort for this is quite high and therefore we do not follow this at the moment.

**Reply by ArminArmout** [reply]
Thank you for the quick and clear answer.

**Comment by pos-ei-don** [comment]
This  reminds me that I need to read the system pressure of my heating system.. Interesting Feature request!

**Comment by jomjol** [comment]
To be clear: I don't expect, this to be implemented. Effort is really high.
